package.json

```diff
@@ -21,5 +22,7 @@
     "sass": "^1.26.3",
     "typescript": "^3.8.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "hyperscript": "^2.0.2"
```

Comment:
HTML-in-JS without a library becomes really tedious, this is the least of evils :P